Feature/new qos [5482] (#4)
eProsima/Shapes-Demo-Docs: docs/firststept.rst

- **History and Durability:** The History of the Publishers is set to **KEEP_LAST**. The user can select the number of samples that the Publisher is going to save. The user can also select whether this History is going to be **VOLATILE** or **TRANSIENT_LOCAL**. The latter will send that last stored values to subscribers joining after the Publisher has been created.

- **Liveliness:** The user can select the Liveliness Qos for the Publisher between three different values: **AUTOMATIC**, **MANUAL_BY_PARTICIPANT** and **MANUAL_BY_TOPIC**. The user can also set the Lease Duration for the Publisher.

- **Ownership:** The Ownership Qos determines whether the instance (color) of the Topic (Shape) is owned by a single Publisher. If the selected ownership is **EXCLUSIVE** the Publisher will use the Ownership strength value as the strength of its publication. Only the publisher with the highest strength can publish in the same Topic with the same Key.

- **Deadline:** The Deadline Qos determines the maximum expected amount of time between samples.

- **Lifespan:** The Lifespan Qos determines the duration while the sample is still valid.

> - **Disable Positive ACKs:** If enabled, doesn't expects for positive ACKs from the subscriber, considering delivered sent samples after the indicated duration if no negative ACKNACK was received.

Reviewer: ```suggestion
- **Disable Positive ACKs:** If enabled, writers don't expect positive ACKNACK messages from the subscriber. Instead they consider samples to be delivered after the indicated duration if no negative ACKNACK was received.
```

> Take into acoount that Disabling positive ACKs and modify Lifespan will not have any visual effect.

Reviewer: ```suggestion
Note that using Disable Positive ACKs and Lifespan QoS will not have any visual effect.
```

This menu is very similar to the Publication menu but the user cannot change the color and size, and it has additional elements:

- **Time Based Filter:** This value can be used by the user to only specify the minimum amount of time (in milliseconds) that the subscriber wants between updates.

- **Content Based Filter:** This filter draws a rectangle in the shapes window. Only the instances that are included in this rectangle are accepted. The rest of them are ignored. The user can dynamically resize and move this content filter.

> 

Reviewer: Sorry I missed this in my previous review, but `Deadline` and `Lifespan` should also be added here as they also apply to the subscriber side
